src/plugins/intel_gpu/src/kernel_selector/tensor_type.cpp

```diff
@@ -550,6 +550,23 @@ DataTensor DataTensor::FlattenEverything() const {
     return res;
 }
 
+void DataTensor::SwapXY() {
+    DataLayout l = Tensor::bfyx;
+
+    auto x = X();
+    auto y = Y();
+
+    // Swap XY axes.
```

Comment:
I think it would be better to have some assertion code here.

Reply:
Added